### PR TITLE
feat: order list in primeNg

### PR DESCRIPTION
**Pull request by jaspret1**
## Change Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/dispatch/orders/orders-list/orders-list.component.ts [hunk 1]
```diff
@@ -336,6 +373,77 @@ export class OrdersListComponent implements OnInit {
...
+              res.amount = ''
+              res.pickupLocData = ''
+              res.dropLocData = ''
+              res.customerData = ''
+              for (let shipArr of res.shippersReceiversInfo) {
+                for (let ship of shipArr.shippers) {
+                  for (let shipPickUp of ship.pickupPoint) {
+                    let shipDate = shipPickUp
+                    for (let commoD of shipDate.commodity) {
+                      res.commodityData = 'Name:- ' + commoD.name + '\n'
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/orders/orders-list/orders-list.component.ts]: <strong>Name</strong>

File: src/app/pages/dispatch/orders/orders-list/orders-list.component.html [hunk 2]
```diff
@@ -1,1804 +1,291 @@
...
+                          {{ order.createdDate | date: "yyyy/MM/dd" }}
+                        </td>
+                        <td routerLink='/dispatch/orders/detail/{{order.orderID}}' style="width:9%">
+                          <div style="white-space: pre-line;">
+                            {{order.customerData}}
+                          </div>
+                        </td>
+                        <td routerLink='/dispatch/orders/detail/{{order.orderID}}' style="width:15%">
+                          <div style="white-space: pre-line;">
+                            {{order.pickupLocData}}
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/orders/orders-list/orders-list.component.html]: < [innerHTML]="order.pickupLocData"></div>

https://stackoverflow.com/questions/49013217/how-to-render-string-with-html-tags-in-angular-4

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Refer comments

File: src/app/pages/dispatch/orders/orders-list/orders-list.component.html (hunk 2 above)
Comment (jaspret1) [reply src/app/pages/dispatch/orders/orders-list/orders-list.component.html]: done bhaji

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**